src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -15,6 +15,10 @@
 import seedu.address.model.module.ZoomLink;
 import seedu.address.model.module.grade.Assignment;
 import seedu.address.model.tag.Tag;
+import seedu.address.model.task.Criterion;
+import seedu.address.model.task.Date;
```

Comment:
Perhaps we can shift this Date class into a commons package in the model package. The rationale is that other implementations such as our schedule feature would require a Date class as well. This can prevent repetition of code.

Reply:
Agreed, I also think the same.